tests/examples/test_wds_e2e.py

```diff
@@ -96,35 +99,45 @@ def test_wds(catalog, webdataset_tars):
 
 
 def test_wds_merge_with_parquet_meta(catalog, webdataset_tars, webdataset_metadata):
-    wds = DataChain.from_storage(Path(webdataset_tars).as_uri()).gen(
-        laion=process_webdataset(spec=WDSLaion), params="file"
-    )
+    with warnings.catch_warnings():
+        # Ignore SQLAlchemy warnings about cartesian products as these warnings are
+        # not accurate, and are generated due to a bug in ClickHouse-SQLAlchemy.
+        # All queries do have ON clauses, and this warning is not generated in SQLite.
+        if os.environ.get("DATACHAIN_METASTORE") or os.environ.get(
```

Comment:
Q: why this `or` ? is there a better way to detect that CH version is used ... e.g. run some kind of instance of ?